packages/clay-navigation/stories/index.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayNavigation from '../src';
+import React from 'react';
+import {number} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/react';
+
+const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
+
+storiesOf('ClayNavigation', module)
+	.add('default', () => (
```

Comment:
You can rename it to `/w breadcrumb`? Since we can have several other components here.

Reply:
makes sense. Maybe in the future, we can create a folder here.